ui/app/manager/src/index.ts

```diff
@@ -48,6 +48,7 @@ import { ManagerAppConfig } from "@openremote/model";
 import {pageGatewayTunnelProvider} from "./pages/page-gateway-tunnel";
 
 declare var CONFIG_URL_PREFIX: string;
+declare var MANAGER_URL: string;
```

Comment:
Can be `undefined` good practice to declare all possible scenarios so type checking can make sure you handle the different situations